README.md

```diff
@@ -77,6 +77,12 @@ simply run the following from the repo root:
 ./scripts/schema.sh
 ```
 
+## Testing
+
+Run `cargo t` or `cargo unit-test` from the project root to run the unit tests.
```

Comment:
Also, since the `SKIP_CONTRACT_STORE=true` is flakey because of the need to have the correct contract_ids stored, Im going to just recommend people not use that locally. If they just want to simply run all of the integration tests locally  then we should show them the most reliable way.

Reply:
Also, to be clear the line you commented on was for the unit tests, I assume you were talking about the line below about pointing to the integration test readme.